service/prometheus/scrapeconfig.go

```diff
@@ -41,53 +41,77 @@ func getTarget(service v1.Service) model.LabelSet {
 // getScrapeConfig takes a Service, and returns a ScrapeConfig.
 // It is assumed that filtering has already taken place, and the cluster annotation exists.
 func getScrapeConfig(service v1.Service, certificateDirectory string) config.ScrapeConfig {
-	clusterID := service.ObjectMeta.Annotations[ClusterAnnotation]
+	clusterID := GetClusterID(service)
```

Comment:
Other operators have such things in a key package. Note this is also versioned like `keyv2`.

Reply:
Yeah, this could use refactoring.